README.md

```diff
@@ -47,6 +47,37 @@ mvn package
 and then copying and unzipping the zip archive generated in the `target/components/packages`
 directory onto the plugin path or classpath for your Kafka Connect worker(s).
 
+## Running standalone in docker
+
+To run this in standalone-mode via a docker container. (This brings up the environment + the connect-standalone):
+
+```bash
+#Build the project 
+mvn clean package
+
+# Build the docker container 
+docker build -t redditconnector .
+
+# Run 
+docker run --env-file reddit.env -it redditconnector
+
+```
+
+The environment file that was used looks like this: (To configure the subreddit, comment location, and where your broker can be reached)
+
+```bash
+SUBREDDIT=chicagohelicopters
+COMMENTS_SUBREDDIT=chicagohelicopters
+KAFKA_SERVER=<kafka hostname>:9092
+```
+
+Note: Change the kafka host name. If the variable in the file or docker run arguements is not present then it will default to the following values:
```

Comment:
Nit: spelling, wording
```suggestion
Note: Make sure to change the Kafka host name. Variables not specified in the env file or docker run arguments will default to the following values:
```

Reply:
Also, might be worth pointing out that people can use the `docker.for.mac.localhost` hostname if they're using Mac and have a Kafka broker running locally.